estuary/models/errata.py

```diff
@@ -15,6 +15,11 @@
 class Advisory(EstuaryStructuredNode):
     """Definition of an Errata advisory in Neo4j."""
 
+    class BuildAttachedRel(StructuredRel):
+        """Definition of a relationship between another artifact and a KojiBuild added to it."""
+
+        time_added = DateTimeProperty()
```

Comment:
Could you call this just `time` or `time_attached`? I know the UMB messages use the term added, but we're already down the road of using the term attached in this code base.